meteor-app/imports/test/fake-part.js

```diff
@@ -0,0 +1,30 @@
+import { Meteor } from 'meteor/meteor' // base
+import Parts from '/imports/api/parts/schema'
+import casual from 'casual'
```

Comment:
the `casual` library doesn't play nice with Storybook. The reason is that webpack doesn't support dynamic require statements which are used by casual. Try `fakerjs` instead https://github.com/marak/Faker.js/